README.md

```diff
@@ -13,3 +13,81 @@ which is then inserted into the HoloPortOS instance.  When the device boots, it
 - Start the Holo services
 - Eject the USB and blacklist the kernel modules
 
+## Building & Geneating a `HoloPortConfiguration`
+
+We'll generate a `HoloPortConfiguration` object in JSON form, into `holo.json`:
+
+```
+$ nix-build -A holo-configure
+$ ./result/bin/holo-configure --name "HP1" --email "a@b.ca" --password "secret" | tee holo.json
+```
+
+If that doesn't work, try using the nix-shell and manual build approach:
+```
+$ nix-shell
+$ cargo build --release --lib --bin holo-configure
+
+$ ./target/release/holo-configure --name "HP1" --email "a@b.ca" --password "secret" | tee holo.json
+Generating HoloPort Configuration for email: a@b.ca
+{
+  "name": "HP1",
+  "email": "a@b.ca",
+  "admin_pubkey": "HcACj5GG78Nfa476fdvcAXwOQdv7hq7gyHi6bueg7ZSb4iix5hNpUcDFjnjejvi",
+  "seed_key": "HcBciRfTSA9E9h6y9mdfzi95PPYhkj6qfBHsXRAC34hpqjx3kxHNIWKYzexuzva",
+  "seed": "HcCcjf5ebsEDxmiz8j7YZX3r47qriFhwwPFZc7Vc7Wys5ozcFwUAQQTXoaygnbr",
+  "seed_sig": "WO0PYkFg1RZEP1UOzdBacj5QtHuM37uqjn0zPSSsgw8gJX2TU4NoQNb3tDMNvSFK5n4dDcen10ScGsRIde5iCA=="
+}
+```
+
+### `name`
+
+Optionally, make the `admin_key` and `seed_key` unique, by hashing the supplied `name` into
+`password` when performing the Argon2 password hashing.
```

Comment:
`admin_key` should be unique by using derived Holochain public key as Argon2 `secret`. There shouldn't be a `name`.

Reply:
Ah; right; we agreed to use the seed as the "unique" aspect of each config...  OK, I'll remove name, and use the seed then (either directly, or indirectly via first deriving the Holo Agent ID, and then deriving it from the Agent ID private key).  Yes, I forgot about our agreement to do that.